src/alert/alert.scss

```diff
@@ -17,43 +18,53 @@
     background-color: t(iui-color-background-1);
   }
 
-  &-icon {
-    @include iui-icons-default;
-    flex-shrink: 0;
-    margin-left: $iui-m;
-  }
-
-  &-message {
-    margin: $iui-baseline $iui-m;
-  }
-
-  &-link {
-    @include iui-anchor-underline('on-initial');
-    border-radius: $iui-border-radius;
-    cursor: pointer;
-    margin-left: $iui-s;
-    user-select: none;
-    white-space: nowrap;
-  }
-
-  > .iui-button {
-    margin-left: auto;
-    margin-right: $iui-s;
-  }
-
   &.iui-sticky {
     border-radius: 0;
-    position: absolute;
+    position: fixed;
     top: 0;
     left: 0;
     width: 100vw;
```

Comment:
Expanding upon the comment above this one, if we do allow an alert nested within another component like a modal, I think this `width: 100vw` may cause issues.  `100%` might make more sense.

Reply:
Yeah, so I guess lets use `%` for width OR can test out having `display: flex` with idea that it will stretch.
Then we need to think about `position` here..

Reply:
`sticky` might be an option as it is relative to the first scrolling parent.

Reply:
`display: flex` and `position: sticky` have been added here

Reply:
`display: flex` is not needed, cause its coming from `iui-alert`

Reply:
Removed